uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/FuseOnOpenShiftTest.java

```diff
@@ -144,7 +144,9 @@ public void testDeploymentToOpenShift() {
 		// check the deployment
 		explorer.activate();
 		explorer.openPodLog(OPENSHIFT_CONNECTION, OPENSHIFT_PROJECT_NAME, OPENSHIFT_DEPLOYMENT);
-		new WaitUntil(new ConsoleHasText(DEPLOYMENT_OK), TimePeriod.VERY_LONG);
+		new WaitUntil(new ConsoleHasText(DEPLOYMENT_OK), TimePeriod.VERY_LONG, false);
+		explorer.activate();
+		explorer.openPodLog(OPENSHIFT_CONNECTION, OPENSHIFT_PROJECT_NAME, OPENSHIFT_DEPLOYMENT);
```

Comment:
sometimes the console has different output - not the actual pod log, but status of the pod. This is my workaround :-)